Update IYS2022_Shimada_Data_template.xlsx

shaunwbell opened the pull request:
- add a field for profile number
- embed profile number in Station_Event_ID
- add description in Cruise Instrument and Data Dictionary

shaunwbell committed:
Update IYS2022_Shimada_Data_template.xlsx

- add a field for profile number
- embed profile number in Station_Event_ID
- add description in Cruise Instrument and Data Dictionary

Br-Johnson commented:
Thanks @shaunwbell. The addition to the Station Event ID in the CTD INFO table is good. Just a couple of things needed to finish this up:

- The Station_Event_ID field in the table '4. SAMPLING EVENT INFO' also needs to have the profile ID added to the event ID and should match exactly to the Station_Event_ID in the table '7. CTD INFO'. The way it is right now I'm unable to join the SAMPLING EVENT INFO to CTD INFO and so I can't  get locations and times for each CTD cast for example.
- Add a 'Profile_Number' column to the SAMPLING EVENT INFO table between 'Station' and 'Event_Type'
- Change the column name in CTD INFO back to 'Station_Event_ID' rather than 'Station_Event_Profile_ID' so that the column names match between the two tables. 
- Keep 'Shimada' lower case in the CTD INFO Station_Event_ID to match the structure in SAMPLING EVENT INFO
- Add '2022' in the Station_Event_ID in the CTD info table. Ie. 'IYS2022-Shimada-4-1-001-CTD'

shaunwbell committed:
Update IYS2022_Shimada_Data_template.xlsx

- addressed event id syntax, corrected inconsistencies across tables.
- Corrected one scientific sampling event that had been omitted from the initial upload

shaunwbell commented:
@Br-Johnson 

I believe I addressed all your concerns.  Thanks for spelling out the challenges - super helpful!

Br-Johnson commented:
Looks perfect. Thanks!
